source/synthDrivers/oneCore.py

```diff
@@ -481,13 +505,10 @@ def _getDefaultVoice(self):
 			for voice in voices.values():
 				if voice.language.startswith(baseLanguage):
 					return voice.id
-		# Just use the first available

```

Comment:
Why this change? Using the first available voice seems to me to be more preferable to switching to eSpeak. I can also imagine a situation in which there is no OneCore voice for whatever language is the Windows default one but there is a voice for a language close enough in pronunciation that it  can easily be used so this is a change in behavior.

Reply:
This is the cause of #10451. Of course, one may want to set a OneCore voice to override espeak, but when checking this for the first time (eg installing NVDA) we need to provide a voice that we can be sure won't silently fail.